utils/components-testkit/src/main/scala/pl/touk/nussknacker/engine/util/test/TestScenarioRunner.scala

```diff
@@ -13,5 +21,20 @@ import scala.reflect.ClassTag
 trait TestScenarioRunner
 
 trait ClassBaseTestScenarioRunner extends TestScenarioRunner {
-  def runWithData[T:ClassTag, R](scenario: EspProcess, data: List[T]): List[R]
+  def runWithData[T:ClassTag, R](scenario: EspProcess, data: List[T]): RunnerResult[R]
```

Comment:
Can u change class name to `ClassBasedTestScenarioRunner`? I missed it during previous review